### PR TITLE
Aventus Storage Unit Test scripts

### PR DESCRIPTION
**Pull request by babasholaking**
Aventus Storage Unit Test scripts

### CODE REVIEW

File: test/AventusStorageTest.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Specifically request an abstraction for AventusStorage
+const AventusStorage = artifacts.require("AventusStorage.sol");
+
+contract('AventusStorage', function () {
+    var avtStorage;
+    var owner = "0x627306090abaB3A6e1400e9345bC60c78a8BEf57";
```
Comment (andy-mcewan-aventus-io) [inline test/AventusStorageTest.js]: can delete owner var now

File: test/AventusStorageTest.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        var value = await avtStorage.getInt("BobKing");
+        assert.equal(value, 1234, "Expected value returned from storage");
+
+        await avtStorage.deleteInt("BobKing");
+        value = await avtStorage.getInt("BobKing");
+        assert.equal(value, 0, "Zero returned from storage!");
+    });
+
+    it("can store and retrieve Boolean from AventusStorage", async function() {
+        await avtStorage.setBoolean("BobKing", false);
```
Comment (andy-mcewan-aventus-io) [inline test/AventusStorageTest.js]: use true here as the default value for a key that doesn't exist is false

File: test/AventusStorageTest.js (hunk 1 above)
Comment (babasholaking) [reply test/AventusStorageTest.js]: Deleted...

File: test/AventusStorageTest.js (hunk 2 above)
Comment (babasholaking) [reply test/AventusStorageTest.js]: Updated to use true

**Comment by vtan8** [comment]
Apart from the edge cases, you may want to test some implicitly convertible cases, eg empty string or null against string, a 2 against bool etc.
